test/Client.test.ts

```diff
@@ -291,3 +291,37 @@ describe('ClientOptions', () => {
     })
   })
 })
+
+describe('VoodooClient', () => {
```

Comment:
nit: probably want a separate test file for the voodoo stuff.

Reply:
Good call, I'll move it out into a separate file